src/components/List/List.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import styled from 'styled-components';
+import { typography } from '../../shared';
+import { IconButton } from '../IconButton/IconButton';
+import { MdOutlineExpandLess, MdOutlineExpandMore } from 'react-icons/md';
+import { listItemPadding } from './styles';
+
+/**
+ * ListItem component properties
+ * Extends html label attributes
+ * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes
+ */
+export interface ListItemProps
+  extends React.LabelHTMLAttributes<HTMLLabelElement> {
+  /**
+   * Text of the list item
+   */
+  text?: string;
+
+  /**
+   * Does the item have a button for expanding sub items
+   */
+  expandButton?: boolean;
+
+  /**
+   * Aria-label for the expand button
+   */
+  expandButtonAriaLabel?: string;
+
+  /**
+   * Is the list item by default expanded (true) or unexpanded (false)
+   */
+  expanded?: boolean;
+}
+
+/**
+ * Style for the area which contains the list item's text
+ */
+const TextWrapper = styled.div`
+  font-size: ${typography.size.paragraph};
+  line-height: ${typography.lineHeight.paragraph};
+  flex: 1;
+  display: inline-block;
+  padding: ${listItemPadding};
+`;
+
+/**
+ * Style for the area which contains the list item's expand button
+ */
+const ExpandButtonWrapper = styled.div`
+  padding-right: ${listItemPadding};
+  flex: 1;
+  display: inline-block;
+`;
+
+/**
+ * Style for the list item div, which contains the text and the expand button
+ */
+const ListItemWrapper = styled.div`
```

Comment:
Näissä pitäis varmaan olla cursor: pointer, ettei tulis tota tekstin maalaamis cursoria